functions/src/db/isTokenValidInDb.ts

```diff
@@ -0,0 +1,23 @@
+import { JWTData } from '../types/JWTData'
+import { JWTToken } from '../types/JWTToken'
+import { db } from '../firebase/firebase'
+import { UserData } from '../types/UserData'
+
+export const isTokenValidInDb = async (
+    jwtData: JWTData,
+    token: JWTToken
+): Promise<boolean> => {
+    const result = await db.collection('users').doc(jwtData.uid).get()
```

Comment:
Typing?

Reply:
it's infered with the tyeps from Firestore.